File: Completed/js/areSimilar/areSimilar.py
def areSimilar(a, b):
    flag = False
    for i in range(len(a)):
        if(a[i] != b[i]):
            if(i == len(a) or flag):
                return False
            if(a[i+1] == b[i]):
                temp = a[i]
                a[i] = a[i+1]
                a[i+1] = temp
            elif(a[i] == b[i+1]):
                temp = b[i]
                b[i] = b[i+1]
                b[i+1] = temp
            flag = True
    return True


print(areSimilar([1, 2, 3], [1, 2, 3]))
print(areSimilar([1, 2, 3], [2, 1, 3]))
print(areSimilar([1, 2, 2], [2, 1, 1]))
print(areSimilar([1, 3, 2, 1], [3, 1, 2, 1]))
print(areSimilar([1, 3, 2, 1], [3, 1, 1, 2]))

File: Completed/js/areSimilar/areSimilar.js
function areSimilar(a, b) {
    let flag = false,
        len = a.length;
    for (let i = 0; i < len; i++) {
        if (a[i] != b[i]) {
            if (i == len || flag) {
                return false
            }
            if (a[i + 1] == b[i]) {
                swap(a, i, i + 1);
            } else if (a[i] == b[i + 1]) {
                swap(b, i, i + 1);
            }
            flag = true;
        }
    }
    return true
}
let swap = (a, i, j) => {
    let temp = a[i];
    a[i] = a[j];
    a[j] = temp;
}

console.log(areSimilar([1, 2, 3], [1, 2, 3]));
console.log(areSimilar([1, 2, 3], [2, 1, 3]));
console.log(areSimilar([1, 2, 2], [2, 1, 1]));
console.log(areSimilar([1, 3, 2, 1], [3, 1, 2, 1]));
console.log(areSimilar([1, 3, 2, 1], [3, 1, 1, 2]));